index.html

```diff
@@ -376,113 +396,203 @@ <h4>Characters</h4>
-          <p>It MAY have a <dfn>Text Styles</dfn> property, which provides styles associated with the <a>Text</a> of this <a>Event</a>.</p>
+          <p>The <dfn>Text</dfn> object is used in the <a>Main Text</a> property or the <a>Contextual Text</a> property of an <a>Event</a> object to provide text content in a single language, to document this language and provide optional styling for the text content.</p>
           <p class="issue" data-number="15"></p>
-          <p>It MAY have a <dfn>Text Language</dfn> property. If not present, the <a>Text</a> is assumed to be in the Script <a>Primary Language</a>.</p>
-          <p class="note">In <a>Original Language Dialogue List</a>, <a>Events</a> may be in different languages.</p>
-        </section>
-
-        <section>
-          <h4>Contextual Text</h4>
-          <p>Each <a>Event</a> MAY have a <dfn>Contextual Text</dfn> property, which is a representation of the <a>Text</a> in a different language. It may be used to assist in the processing of the <a>Text</a>.
-          <p class="note">For example, when a <a>Pre-Recording Dub Script</a> needs to be adjusted, having this additional text in the original language(s) could be useful.</p>
-          <p class="issue" data-number="16"></p>
-          <p class="issue" data-number="21"></p>
-          <p>It MAY have a <dfn>Contextual Text Styles</dfn> property, which provides styles associated with the <a>Contextual Text</a> of this <a>Event</a>.</p>
-          <p class="issue" data-number="15"></p>
-          <p>It MAY have a <dfn>Contextual Text Language</dfn> property, representing the language of the <a>Contextual Text</a> content.</p>
-        </section>
-
-        <section>
-          <h4>Event Description</h4>
-          <p>Each <a>Event</a> MAY have an <dfn>Event Description</dfn> property, as a human-readable description of the <a>Event</a>. </p>
-          <p class="issue" data-number="30"></p>
-          <p class="note">It does not have to be unique for this <a>Event</a>. It may be used to identify in a human-readable way one or more <a>Events</a> that need to be processed together, e.g. in a batch recording.</p>
-          <p class="issue" data-number="20"></p>
+          <p>A <a>Text</a> object is represented with a <code>p</code> element with the following constraints:
+            <ul>
+              <li>The text content of the <code>p</code> is the <a>Text</a> of the <a>Event</a>. It MAY also use TTML markup to represent styles.
+                <div class="note">The text content of the paragraph may contain TTML elements such as <code>span</code> or TTML attributes such as <code>tts:ruby</code> used to alter the layout or styling of each paragraph</div>
+              </li>
+              <li>The <code>p</code> element MUST have an <code>xml:lang</code> attribute corresponding to the language of the <a>Text</a> object.</li>
```

Comment:
What if it is the same as the `xml:lang` of the parent? Is it always required?

Reply:
yes always required, for simplicity. The parent would not have it.

Reply:
Not sure I'm happy with this: it's subverting the semantics of `xml:lang` which allow for inheritance of language down the tree. One for a discussion I think.

Reply:
#62